shipment_advice/models/stock_dock.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class StockDock(models.Model):
+    _name = "stock.dock"
```

Comment:
@sebalix Shouldn't this be in a separate module ?

Reply:
Indeed, I will do that